src/app/component/ExpandableContent/ExpandableContent.component.js

```diff
@@ -102,7 +115,10 @@ export class ExpandableContent extends PureComponent {
         }
         this.setState(
             ({ isContentExpanded }) => ({ isContentExpanded: !isContentExpanded }),
-            () => this.scrollToExpandedContent(['NavigationTabs', 'ProductActionsWrapper', 'CartPageSummary'])
+            () => this.scrollToExpandedContent(
+                ['NavigationTabs', 'ProductActionsWrapper', 'CartPageSummary'],
```

Comment:
I think hard-coding this here is not the best idea. Can we pass this from outside?

Reply:
Not quite sure, as these elements only difference is that they have a fixed position. If those blocks would be inside main fixed elements like footer then we could.
So would need to adjust those elements to be rendered inside that main fixed element or create some logic which stores top and bottom fixed element id inside the store.